example/spec/parallel/lzardsBackup/LzardsBackupSpec.js

```diff
@@ -172,27 +176,61 @@ describe('The Lzards Backup Task ', () => {
-    else {
-      expect(lzardsBackupOutput.FunctionError).toBe(undefined);
-    }
-  });
+  describe('The Lzards Backup Task with override configured', () => {
+    it('invokes successfully', async () => {
+      const taskConfig = {
+        lzardsProvider: 'BOGUS_PROVIDER',
+        cumulus_message: {
+          outputs: [
+            {
+              source: '{$.originalPayload}',
+              destination: '{$.payload}',
+            },
+            {
+              source: '{$.backupResults}',
+              destination: '{$.meta.backupStatus}',
+            },
+          ],
+        },
+      };
 
-  it('has the expected backup information', () => {
-    const backupStatus = JSON.parse(new TextDecoder('utf-8').decode(lzardsBackupOutput.Payload)).meta.backupStatus;
-    console.log(`backupStatus: ${JSON.stringify(backupStatus)}`);
-    expect(backupStatus[0].status).toBe('COMPLETED');
-    expect(backupStatus[0].statusCode).toBe(201);
-    expect(backupStatus[0].granuleId).toBe(granuleId);
-    expect(backupStatus[0].provider).toBe(provider);
-    expect(backupStatus[0].createdAt).toBe(tenMinutesAgo);
-    expect(backupStatus[0].collectionId).toBe(encodedConstructCollectionId(collection.name, collection.version));
+      await testSetup({ task_config: taskConfig });
+      if (beforeAllFailed) fail('beforeAll() failed');
+      else {
+        expect(lzardsBackupOutput.FunctionError).toBe(undefined);
+      }
+    });
+
+    it('has the expected backup information', () => {
+      const backupStatus = JSON.parse(new TextDecoder('utf-8').decode(lzardsBackupOutput.Payload)).meta.backupStatus;
+      console.log(`backupStatus: ${JSON.stringify(backupStatus)}`);
+      expect(backupStatus[0].status).toBe('FAILED');
+      expect(backupStatus[0].body).toContain('Unprocessable Entity');
+    });
   });
 
-  describe('The Lzards API Client', () => {
+  describe('The Lzards Backup Task', () => {
+    it('invokes successfully', async () => {
+      await testSetup();
+      if (beforeAllFailed) fail('beforeAll() failed');
+      else {
+        expect(lzardsBackupOutput.FunctionError).toBe(undefined);
+      }
+    });
+
+    it('has the expected backup information', () => {
+      const backupStatus = JSON.parse(new TextDecoder('utf-8').decode(lzardsBackupOutput.Payload)).meta.backupStatus;
+      console.log(`backupStatus: ${JSON.stringify(backupStatus)}`);
+      expect(backupStatus[0].status).toBe('COMPLETED');
+      expect(backupStatus[0].statusCode).toBe(201);
+      expect(backupStatus[0].granuleId).toBe(granuleId);
+      expect(backupStatus[0].provider).toBe(provider);
+      expect(backupStatus[0].createdAt).toBe(tenMinutesAgo);
+      expect(backupStatus[0].collectionId).toBe(encodedConstructCollectionId(collection.name, collection.version));
+    });
+
     it('throws an error when no search parameters are provided', async () => {
       if (beforeAllFailed) fail('beforeAll() failed');
```

Comment:
So here we check `beforeAllFailed` but don't do the test setup inside of this test. We're relying on another previous test calling `await testSetup();`? I'm not sure about that. These aren't serial tests so I don't think we can guarantee that happens?

Reply:
I don't believe that's what's happening. 

`let beforeAllFailed = false;` is set at the top of all tests. 

testSetup is run initially in both sub-stanzas, and hits the 

```javascript
    } catch (error) {
      beforeAllFailed = true;
      throw error;
    }
  };
  ```
  
  block if it throws on either invocation of that function.
  
  You are right, if the first test fails, the second won't bother evaluating.    Given the goal is to not wait evaluating test outputs if we know it didn't setup right, and this is an invocation per sub-test stanza..... 
  
How about: e66c094b20

Reply:
Yeah I think that makes sense. I see your point about defaulting the `beforeAllFailed` flag. You're right that this isn't really a problem. Change makes sense though.